src/main/java/com/znsio/teswiz/runner/Driver.java

```diff
@@ -439,12 +439,10 @@ public boolean isDriverRunningInHeadlessMode() {
     }
 
     public WebDriver setWebViewContext() {
-// todo - to be fixed for appium 2.0
-//        AppiumDriver appiumDriver = (AppiumDriver) driver;
-//        Set<String> contextNames = appiumDriver.getContextHandles();
-//        return appiumDriver.context((String) contextNames.toArray()[contextNames.size() - 1]);
+        LOGGER.info("Setting web view context");
         SupportsContextSwitching contextSwitchingDriver = (SupportsContextSwitching) driver;
         Set<String> contextHandles = contextSwitchingDriver.getContextHandles();
+        contextHandles.stream().forEach(handle -> LOGGER.info("Value of context handle " + handle));
```

Comment:
What does the message "Valud of context handle ...." mean? Rephrase it

Reply:
updated